certbot-apache/certbot_apache/constants.py

```diff
@@ -62,9 +62,9 @@
     vhost_files="*.conf",
     logs_root="/var/log/apache2",
     version_cmd=['/usr/sbin/apache2', '-v'],
-    define_cmd=['apache2ctl', 'virtualhosts'],
-    restart_cmd=['apache2ctl', 'graceful'],
-    conftest_cmd=['apache2ctl', 'configtest'],
+    define_cmd=['/usr/sbin/apache2', '-S'],
```

Comment:
Still the current status is, that only OpenRC users can use certbot (with the apache plugin) and systemd users can't.
Whereas with this change both can use it and just the autoconfigure feature isn't available to a few of them.

I'm open for alternatives that have equal benefits but lesser disadvantages.